docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+- This is a good time to configure and check the operation of any flight modes and switches you might want.
+
+## 9. Bench and range testing
+
+Before your first flight with mLRS, it is good to do some additional testing to ensure everything is working as expected.
+
+- Remove props and power everything up.
+- Verify the Rx and Tx modules connect by observing the LEDs.
+- Use the mLRS Lua script or CLI to set the Rx and Tx power to the lowest setting.
+- Start your ground station and verify that parameters download in a reasonable time depending on mode (6-60 seconds).
+- Verify that everything you expect such as attitude, GPS position, battery voltage, etc is being displayed correctly by both the ground station and Yaapu telemetry if you use it.  If you don't use Yaapu or don't see what you expect, check that you see the expected telemetry sensors on your radio (MDL -> TELEMETRY -> Discover new sensors)
+- Ensure you can change flight modes and arm/disarm the aircraft from both your RC radio and your ground station.
+- Perform a "walking range test" by leaving the aircraft behind while you walk away with the transmitter and ground station.
+- If mLRS doesn't work as expected, see the [Troubleshooting](TROUBLE.md) section.
+- Don't forget to set the Rx and Tx power back to a reasonable level before your first flight.
+
+## 10. Fail-safe configuration
+
+Check and test your failsafe behavior and configuration ***before*** you experience a connection loss in flight.
+
+- You may choose to configure ArduPilot for failsafe actions for loss of either MAVLink telemetry or radio control or both.
+- Note that even though both MAVLink and RC are transported by mLRS in the same radio frames, the design is such that loss of radio control may occur at a slightly longer range than loss of telemetry at the receiver.
+- Note also that, of course, loss of MAVLink telemetry messages at the ground station may not occur at the same range as loss of MAVLink telemetry messages at the receiver, depending on the transmit power and RF conditions at either end of the radio link.  Thus, your ground station may report "communication lost" before or after ArduPilot initiates failsafe actions.
+- The "no signal" method is highly recommended for RC failsafe.
+- See [Rx Failsafe Mode](PARAMETERS.md#rx-failsafe-mode).
+- See ArduCopter [Radio failsafe](https://ardupilot.org/copter/docs/radio-failsafe.html) documentation.
+
+## 11. Optimizing mLRS
+
+The above 10 steps should get you started with mLRS.  If something doesn't work, try the [trouble shooting guide](TROUBLE.md).
+
+Here are some additional things you should consider to improve your experience.
+- Set your ArduPilot [Stream Rate](CRSF.md#stream-rates) parameters.  When you don't use a ground station, this is required to get the messages which radio telemetry sensors and Yaapu need.  Even if you do always use a ground station, this is a good way to optimize the message flow rates for use with mLRS.
```

Comment:
this is IMHO highly missleading, since mLRS will gracefully "optimize the message flow rates" ...

what one does here is to set so to say a lower limit ... and not an upper limit which I suspect most readers will interpret these words. 

I don't have a good wording suggestion from scratch but think this needs improvement

Reply:
If we are going to assume flow control is enabled and working, should we also simplify the table here? https://github.com/brad112358/mLRS-docu/blob/quickstart/docs/CRSF.md#stream-rates

I added this here because John suggested this didn't belong in the Yaapu section which I assumed meant he thought it might be good to do it even when only using a ground station.  

How about if I replace this with " Set your ArduPilot [Stream Rate](CRSF.md#stream-rates) parameters.  When you don't use a ground station, this is required to get the messages which radio telemetry sensors and Yaapu need.  Even if you do always use a ground station, this is a good way to customize the relative update rates to suit your needs."

Reply:
it appears to be in fact the case that for some vehicles AP would not stream per default (without a GCS), and this has created confusion in the past. So I think we need it, and we should give recommendations.

problem with last sentence is that a GCS usually will override these settings

Maybe 

Check your ArduPilot [Stream Rate](https://github.com/olliw42/mLRS-docu/pull/CRSF.md#stream-rates) parameters. When you don't use a ground station, these have to be set to get the MAVLink messages which radio telemetry sensors and Yaapu need.

Reply:
Similar advice was given up above where we discuss Yaapu, but sure, this wording sounds good too.